keyboards/nyquist/keymaps/chaitanya/keymap.c

```diff
@@ -0,0 +1,197 @@
+#include "nyquist.h"
+#include "action_layer.h"
+#include "eeconfig.h"
+
+extern keymap_config_t keymap_config;
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+#define _QWERTY 0
+#define _QWERTY2 1
+#define _ADJUST 8
+#define _ADJUST2 9
+#define _FKeys 16
+
+enum custom_keycodes {
+  QWERTY = SAFE_RANGE,
+  QWERTY2,
+  ADJUST,
+  ADJUST2,
+  FKeys,
+};
+
+// Fillers to make layering more clear
+#define _______ KC_TRNS
```

Comment:
These defines aren't necessary, as these are actually defined in the core.  You can safely remove them. 

https://github.com/qmk/qmk_firmware/blob/a0309db983150fd0197eb620a6ba552d90d29f93/quantum/quantum_keycodes.h#L30-L31

```suggestion
```